src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -14,6 +14,7 @@ import withWindowDimensions, {windowDimensionsPropTypes} from '../../../componen
 import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
 import canUseTouchScreen from '../../../libs/canUseTouchscreen';
 import compose from '../../../libs/compose';
+import colors from '../../../styles/colors';
```

Comment:
use variables instead of colors.

Reply:
I double checked it if colors.js are used in components. And I saw in so many files with this usage.

Do you want me to add a new variable as a theme variable?